onnxruntime/core/providers/cuda/controlflow/if.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include <functional>
+#include "gsl/gsl"
+
+#include "core/common/common.h"
+#include "core/providers/cpu/controlflow/if.h"
+
+namespace onnxruntime {
+class SessionState;
+
+namespace cuda {
+
+// Use the CPU implementation for the logic
```

Comment:
nit: Would it be clearer if the comment said - Use the CPU implementation for the subgraph execution logic ?

Reply:
It's using the CPU implementation for all the logic though. Technically it doesn't even need to override the Compute method.